lib/components/banner/banner.less

```diff
@@ -3,6 +3,7 @@
 // This would reduce the amount of CSS we ship to the client and simplify our codebase
 .s-banner {
     --_no-x-offset: 0;
+    --_no-height: var(--theme-topbar-height, calc(var(--su-static48) + var(--su-static8)));
```

Comment:
It looks like this is a legitimate use case for PPCP. postcss-calc was thrown off by the previous long inlined expression contained in the transform property.

Reply:
Nice catch!

BTW, I renamed these PPCPs to reflect the axis being translated (based on the syntax described [here](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/translate3d#ty)) in [this commit](https://github.com/StackExchange/Stacks/pull/1595/commits/2c80beaa5419218e2d252062d4399fef7c53331b). I think using `x` was a mistake and this commit corrects that and unifies the syntax with the newly-added PPCP.